less/documentDBFabric.less

```diff
@@ -70,6 +70,10 @@ a:focus {
   border-bottom: 2px solid @FabricAccentMedium;
 }
 
+.nav-tabs>li.active>.tabNavContentContainer>.tab_Content>.tabNavText {
```

Comment:
Shouldn't we be having the same element, but just switch color instead?

Reply:
No because we let the parent container have the underline on Fabric, not just the text. It spans the icon and close button as well, making it more consistent with the hover effect.